tools/multi_k8s_dind_cluster_manager.sh

```diff
@@ -101,7 +101,7 @@ function rebuild::deploy_apps {
     hash helm 2>/dev/null
     if [[ $? -eq 0 ]]
     then
-       helm init --tiller-image uhub.ucloud.cn/pingcap/tiller:${TILLER_VERSION}
+       helm init --upgrade || helm init --tiller-image uhub.ucloud.cn/pingcap/tiller:${TILLER_VERSION}
```

Comment:
I guess `helm init --upgrade` would return 0 when it can create tiller deployment. So the second option would never be executed as expected.

Reply:
@onlymellb Can we add an environment variable for GFW.

Reply:
I am working on this, by introducing a environment variable to decide which registry to pull the image from.